config/locales/rdr.en.yml

```diff
@@ -84,7 +84,7 @@ en:
     submissions:
       heading: 'Data Submission Form'
       form_error_html: "We're sorry you've encountered a problem. Please contact <a href='mailto:%{email}'>%{email}</a> for assistance."
-      instructions_text: "Thank you for depositing with the Duke Research Data Repository. When you have finished uploading your files, please send a reply to the email you received confirming your submission, or email us at %{email}. If you have any questions, please review our Submission guidelines or contact us.\n\nResearch data curators will review your deposit to help ensure the data meet the submission requirements and are in a format and structure that supports long-term preservation, access, and reuse. If any changes are needed, you will be contacted. Once the processing is complete, the depositor will be notified and provided with a link to their dataset record and persistent identifier. Our aim is to process all deposits in a timely fashion and turnaround is usually one to three business days, however processing time may vary depending on size and type of data."
+      instructions_text: "Please follow these instructions to finish your deposit with the Duke Research DataRepository:\n\n1. Upload your files by clicking the Upload button in the upper right hand corner or drag-and-drop onto the folder page. You may either upload your files directly to this folder or within sub-folder(s).\n\n2. Let us know when you are done uploading by responding to the email you received with the Box link or email us directly at %{email}.\n\n3. We will be in contact with any questions. We will send you a persistent identifier (DOI) once the files are published.\n\nOur aim is to process all deposits in a timely fashion and turnaround time is usually one to three business days, however processing time may vary depending on size and type of data.\n\nIf you have any questions, contact us at %{email}."
```

Comment:
Looks like a typo here: `DataRepository`

Reply:
Dangit - thanks @seanaery (can't trust copy and paste from Box apparently...)